src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -372,5 +377,41 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
       status2 should be(OK)
       status2.value.instances.filter(_.status == raml.PodInstanceState.Stable) should have size 2
     }
+
+    "deploy a simple pod with unique constraint and then " taggedAs WhenEnvSet(envVar, default = "true") ignore {
+
+      val constraints = Set(
+        Constraint.newBuilder
+        .setField(hostnameField)
+        .setOperator(UNIQUE)
+        .setValue("")
+        .build
+      )
+
+      Given("a pod with a single task")
+      val pod = simplePod("simple-pod-with-unique-constraint", constraints = constraints, instances = 1)
+
+      When("The pod is deployed")
+      val createResult = marathon.createPodV2(pod)
+
+      Then("The pod is created")
+      createResult should be(Created)
+      waitForDeployment(createResult)
+      waitForPod(pod.id)
+
+      When("The pod config is updated")
+      val scaledPod = pod.copy(instances = 2)
+      val updateResult = marathon.updatePod(pod.id, scaledPod, force = true)
+
+      Then("The pod is scaled")
+      updateResult should be(OK)
+      waitForDeployment(updateResult)
```

Comment:
Wait, since the pod can't be scaled up this deployment should never end, correct?